### PR TITLE
[Travis] Running PhpSpec on PHP 7

### PR DESCRIPTION
**Pull request by pamil**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #3312, #3287, #3257, #3260
| License       | MIT

Based on #3287 and #3257.

### CODE REVIEW

File: travis/prepare/mongodb/1-before [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```
Comment (michalmarcinkowski) [inline travis/prepare/mongodb/1-before]: Missing blank line

File: travis/prepare/mongodb/2-rebuild-cache [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+composer require doctrine/mongodb-odm="^1.0.2"
```
Comment (michalmarcinkowski) [inline travis/prepare/mongodb/2-rebuild-cache]: Missing blank line

**Comment by michalmarcinkowski** [comment]
Full compatibility with PHP 7. Nice! :+1:

File: src/Sylius/Bundle/TranslationBundle/spec/Doctrine/ODM/MongoDB/TranslatableResourceRepositorySpec.php [hunk 3]
```diff
@@ -25,6 +25,8 @@
 /**
  * Doctrine ODM MongoDB driver translatable document repository spec.
  *
+ * @require \Doctrine\ODM\MongoDB\DocumentManager
```
Comment (digitalkaoz) [inline src/Sylius/Bundle/TranslationBundle/spec/Doctrine/ODM/MongoDB/TranslatableResourceRepositorySpec.php]: is this `@require FQND` something special for phpspec?

Comment (pamil) [inline src/Sylius/Bundle/TranslationBundle/spec/Doctrine/ODM/MongoDB/TranslatableResourceRepositorySpec.php]: There is no MongoDB extension for PHP 7 so we must skip these specifications that are based on it.

Comment (digitalkaoz) [inline src/Sylius/Bundle/TranslationBundle/spec/Doctrine/ODM/MongoDB/TranslatableResourceRepositorySpec.php]: ah nice. must remind that :)

File: travis/prepare/mongodb/1-before (hunk 1 above)
Comment (pamil) [inline travis/prepare/mongodb/1-before]: @github why does it isn't an outdated diff comment?

File: travis/prepare/mongodb/2-rebuild-cache (hunk 2 above)
Comment (pamil) [inline travis/prepare/mongodb/2-rebuild-cache]: Same here, it should be outdated diff comment :(

**Comment by pjedrzejewski** [comment]
Thank you Kamil! :+1:

**Comment by digitalkaoz** [comment]
nice one :)

**Comment by lchrusciel** [comment]
Good job @pamil  Btw. nice branch name :+1: :dancer:

**Pull request merged**